### PR TITLE
Move frame conversion

### PR DESCRIPTION
**Pull request by brankalakic**

### CODE REVIEW

File: smll/FaceDetector.cpp [hunk 1]
```diff
@@ -129,7 +129,7 @@ namespace smll {
 		}
 	}
 
-	void FaceDetector::DetectFaces(struct obs_source_frame * frame, int width, int height, DetectionResults& results) {
+	void FaceDetector::DetectFaces(cv::Mat inputImage, int width, int height, DetectionResults& results) {
```
Comment (r4ghu) [inline smll/FaceDetector.cpp]: Pass by reference can be a good way to implement here. 
```
void FaceDetector::DetectFaces(cv::Mat& inputImage, int width, int height, DetectionResults& results)
```

File: smll/FaceDetector.cpp [hunk 2]
```diff
@@ -146,7 +146,7 @@ namespace smll {
 		resizeWidth = width;
 		resizeHeight = height;
 
-		ConvertFrameToGrayMat(frame);
+		grayImage = inputImage;
```
Comment (r4ghu) [inline smll/FaceDetector.cpp:149]: Are we still using grayImage in other parts of the code?

**Review by r4ghu — APPROVED** [review]
A general feedback. Changes are approved!

File: smll/FaceDetector.cpp (hunk 1 above)
Comment (brankalakic) [reply smll/FaceDetector.cpp]: Changed it.

File: smll/FaceDetector.cpp (hunk 2 above)
Comment (brankalakic) [reply smll/FaceDetector.cpp:149]: In DetectLandmarks.

**Pull request merged**